schemas/com.snowplowanalytics.snowplow.enrichments/api_lookup_enrichment_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,166 @@
+		},
+		"enabled": {
+			"type": "boolean"
+		},
+
+		"parameters": {
+			"type": "object",
+			"properties": {
+				"inputs": {
+					"type": "array",
+					"items": {
+						"type": "object",
+						"properties": {
+							"key": {
+								"type": "string",
+								"pattern": "^[A-Za-z0-9_-]+$"
+							},
+							"pojo": {
+								"type": "object",
+								"properties": {
+									"field": {
+										"type": "string"
+									}
+								},
+								"additionalProperties": false
+							},
+							"json": {
+								"type": "object",
+								"properties": {
+									"field": {
+										"type": "string",
+										"enum": ["unstruct_event", "contexts", "derived_contexts"]
+									},
+								  "schemaCriterion": {
+									  "type": "string",
+									  "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-([0-9]+|\\*)-([0-9]+|\\*)$"
+								  },
+								  "jsonPath": {
+									  "type": "string"
+								  }
+								},
+								"additionalProperties": false
+							}
+						},
+						"additionalProperties": false,
+						"minProperties": 2,
+						"maxProperties": 2,
+						"required": ["key"]
+					}
+				},
+				"api": {
+					"type": "object",
+					"minProperties": 1,
+					"maxProperties": 1,
+					"properties": {
+						"http": {
+							"type": "object",
+							"properties": {
+								"method": {
+									"type": "string",
+									"enum": ["GET", "POST", "PUT", "DELETE"]
+								},
+								"uri": {
+									"type": "string"
+								},
+								"timeout": {
+									"type": "integer",
+									"minimum": 1,
+									"maximum": 60000
+								},
+								"authentication": {
+									"type": "object",
+									"properties": {
+										"http-basic": {
+											"type": "object",
+											"properties": {
+												"username": {
+													"type": ["string", "null"],
+													"minLength": 1
+												},
+												"password": {
+													"type": ["string", "null"],
+													"minLength": 1
+												}
+											},
+											"minProperties": 1,
+											"additionalProperties": false
+										}
+									},
+									"additionalProperties": false
+								}
+							},
+							"required": ["method", "uri", "timeout", "authentication"],
+							"additionalProperties": false
+						}
+					},
+					"additionalProperties": false
+				},
+				"outputs": {
+					"type": "array",
+					"items": {
+						"type": "object",
+						"properties": {
+							"schema": {
+								"type": "string",
+								"pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$"
+							},
+							"json": {
+								"type": "object",
+								"properties": {
+									"jsonPath": {
+										"type": "string"
+									}
+								},
+								"required": ["jsonPath"],
+								"additionalProperties": false
+							}
+						},
+						"required": ["schema"],
+						"minProperties": 2,
+						"maxProperties": 2,
+						"additionalProperties": false
+					}
```

Comment:
Okay, let's enforce the limit. We can always loosen it with a 1-0-1 in the future.